specification/loadtestservice/LoadTestService/models.testprofile.tsp

```diff
@@ -259,11 +259,11 @@ model ListTestProfileQueryParams {
   lastModifiedEndTime?: utcDateTime;
 
   @doc("Comma separated list of IDs of the test profiles to filter.")
-  @query
-  testProfileIds?: string;
+  @query(#{ explode: false })
+  testProfileIds?: string[];
 
   @doc("Comma separated list IDs of the tests which should be associated with the test profiles to fetch.")
-  @query
+  @query(#{ explode: false })
```

Comment:
do we need to do this for list test and list test runs filters too? For test run status?

Reply:
I initially considered it, but the GA SDK generated off of it will break, hence I did not do it :(